felix/bpf-gpl/conntrack_types.h

```diff
@@ -154,9 +155,9 @@ struct ct_create_ctx {
 	ipv46_addr_t tun_ip; /* is set when the packet arrive through the NP tunnel.
 			* It is also set on the first node when we create the
 			* initial CT entry for the tunneled traffic. */
-	__u16 flags;
+	__u32 flags;
 	__u8 proto;
-	__u8 __pad;
+	__u8 __pad[3];
```

Comment:
Wasn't this supposed to pick ut the changes from master without making a breaking change that would require an upgrade?

Reply:
This is in the create_ctx struct, right? I didn't know that struct had migration code/requirements. We might need a sync-up on this?

Reply:
ohhh yeah that is private, ignore me :+1: